src/bun.js/bindings/ImportMetaObject.cpp

```diff
@@ -269,44 +269,92 @@ JSC_DEFINE_HOST_FUNCTION(functionImportMeta__resolve,
 {
     JSC::VM& vm = globalObject->vm();
 
-    switch (callFrame->argumentCount()) {
-    case 0: {
-        auto scope = DECLARE_THROW_SCOPE(globalObject->vm());
-        // not "requires" because "require" could be confusing
-        JSC::throwTypeError(globalObject, scope, "import.meta.resolve needs 1 argument (a string)"_s);
-        scope.release();
-        return JSC::JSValue::encode(JSC::JSValue {});
-    }
-    default: {
-        JSC::JSValue moduleName = callFrame->argument(0);
+    auto scope = DECLARE_THROW_SCOPE(globalObject->vm());
 
-        if (moduleName.isUndefinedOrNull()) {
+    auto thisValue = callFrame->thisValue();
+    auto specifierValue = callFrame->argument(0);
+    // 1. Set specifier to ? ToString(specifier).
+    auto specifier = specifierValue.toWTFString(globalObject);
+    RETURN_IF_EXCEPTION(scope, JSC::JSValue::encode(JSC::JSValue {}));
+
+    // Node.js allows a second argument for parent
```

Comment:
only behind a flag. it's likely that NodeJS will not support a 2nd argument for `import.meta.resolve` as they want to align with Deno and Browser that dose not support a 2nd argument.

Reply:
I think as of v20.11.1, `import.meta.resolve` supports second argument  (also for what is worth, if it is not supported, tooling support would be hell considering a module cannot resolve on behalf of another)

Reply:
it is still behind a flag: `--experimental-import-meta-resolve`

since we do not have node's feature flags, i think this is ok to just have, but it can just not be present